README.md

```diff
@@ -3034,7 +3034,7 @@ Add the specified import line to all files.
 
 ```toml
 [tool.ruff.isort]
-add-import = ["from __future__ import annotations"]
+required-imports = ["from __future__ import annotations"]
```

Comment:
Oof, thank you! If you have the repo setup, do you mind changing this in `src/isort/settings.rs`, line 136, then running `cargo dev generate-all`?

(If you don't have the repo setup, I'm also happy to do it then push and merge, all good.)

Reply:
Can do later today.

Reply:
I'm making some changes to this file anyway so I got it, all good :)